tests/syscalls.rs

```diff
@@ -0,0 +1,161 @@
+#![deny(warnings)]
+
+use felt::Felt;
+use sha3::{Digest, Keccak256};
+use starknet_rs::{
+    business_logic::{
+        execution::{
+            execution_entry_point::ExecutionEntryPoint,
+            objects::{CallInfo, CallType, OrderedEvent, TransactionExecutionContext},
+        },
+        fact_state::{
+            contract_state::ContractState, in_memory_state_reader::InMemoryStateReader,
+            state::ExecutionResourcesManager,
+        },
+        state::cached_state::CachedState,
+    },
+    definitions::{constants::TRANSACTION_VERSION, general_config::StarknetGeneralConfig},
+    services::api::contract_class::{ContractClass, EntryPointType},
+    starknet_storage::dict_storage::DictStorage,
+    utils::Address,
+};
+use std::path::Path;
+
+fn sn_keccak(entry_point: &str) -> Felt {
```

Comment:
I think @edg-l already implemented sn_keccak in one of his PRs. We should remove this one once it is merged.

Reply:
It's also implemented (privately) within the project.